src/tiles/text-snippet-block.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  nothing,
+  TemplateResult,
+} from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { join } from 'lit/directives/join.js';
+
+@customElement('text-snippet-block')
+export class TextSnippetBlock extends LitElement {
+  @property({ type: Array }) snippets?: string[] = [];
+
+  @property({ type: String }) viewSize: string = 'desktop';
+
+  render() {
+    const viewSizeClass = this.viewSize === 'grid' ? 'grid' : 'list';
+
+    if (!this.snippets || this.snippets.length === 0) return html`${nothing}`;
```

Comment:
👍 :yesgood:

(does `!this.snippets?.length` work?)

Reply:
👍